page-objects/homework.js

```diff
@@ -97,12 +100,12 @@ module.exports = {
 			const containerWithTasks = await driver.$('.col-xl-12');
 			await containerWithTasks.waitForExist(2000);
 			let numOfElems = await containerWithTasks.$$('li');
-			for (var i=1; i<=numOfElems.length-1; i++) {
-					let nameOfTheTaskSelector = await driver.$('.col-xl-12 > li:nth-child('+i+') > .content > h2' );
-					let nameOfTheTask = await nameOfTheTaskSelector.getText();
-					if(await nameOfTheTask.includes(taskname)) {
-						return i;
-					}
+			for (var i = 1; i <= numOfElems.length - 1; i++) {
```

Comment:
```suggestion
			for (var i = 1; i <= numOfElems; i++) {
```
This part may be problematic in original form. As it skips last element.

Reply:
done with next commit